website/i18n/en.json

```diff
@@ -17,16 +17,10 @@
       "build-tools/mill": {
         "title": "Mill"
       },
-      "build-tools/new-build-tool": {
```

Comment:
How come a lot of these are being removed as they still exist on the site?

Reply:
They are no longer under `build-tools`, but under `integrations`, which seems to be the case in all the other changes.

Reply:
Ahh of course. 👍🏼